### DISCUSSION TITLE
Image broken

### DISCUSSION THREAD

**Discussion by Mohanapriya-Vadivel** [community/community#122581] [Programming Help]
I create websit it opens on live server properly ,and I posted it on github and make a site but while visiting site the image in the website are broken any one tell how  will I correct it

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Mohanapriya-Vadivel ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by CopperEagle** [comment]
Hi @Mohanapriya-Vadivel 

It would be great if we could see the code for the website. I did not find it in your repositories.
Typically the path of the image is specified as a local path starting at the project root. Did you do that?
